### PR TITLE
New Bing Cheat Sheet

### PR DESCRIPTION
**Pull request by mahavirrathod**
What does your Instant Answer do?

Shows all the shortcut for bing maps

What is the data source for your Instant Answer? (Provide a link if possible)

http://www.keyboardhelpers.com/bing.html

Why did you choose this data source?

It provides best shortcut for bing maps

Are there any other alternative (better) data sources?

Yes, But this is the official one

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)

Navigators
bing

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?

No

Are you having any problems? Do you need our help with anything?

NO

What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)

### CODE REVIEW

**Comment by daxtheduck** [comment]
@mahavirrathod Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by MariagraziaAlastra** [comment]
Hey @mahavirrathod, thanks for contributing!
Did you know you don't need to fill in the PR template in your first comment anymore? As the comments above say, you just need to create an IA Page [here](https://duck.co/ia/new_ia) - you can just use the link to your PR - and then come back here and paste the link to it in your first comment.
More info about [in the docs](http://docs.duckduckhack.com/submitting/pull-request.html#submitting-a-pull-request).
Thanks!

File: share/goodie/cheat_sheets/json/Bing [hunk 1]
```diff
@@ -0,0 +1,137 @@
+{
+    "id": "Bing_maps_cheat_sheet",
```
Comment (codenirvana) [inline share/goodie/cheat_sheets/json/Bing]: change this to ```bing_maps_cheat_sheet```

**Comment by codenirvana** [comment]
@mahavirrathod rename file name to ```bing-maps.json```

File: share/goodie/cheat_sheets/json/Bing [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+    "description": "Microsoft Bing Maps Keyboard shorcuts", 
+
+    "metadata": { 
+        "sourceName": "Keyboard Helpers",
+        "sourceUrl": "http://www.keyboardhelpers.com/bing.html"
+    },
+    
+  
+
```
Comment (navneetzz) [inline share/goodie/cheat_sheets/json/Bing]: @mahavirrathod there is additional space here remove it.

File: share/goodie/cheat_sheets/json/Bing [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    "metadata": { 
+        "sourceName": "Keyboard Helpers",
+        "sourceUrl": "http://www.keyboardhelpers.com/bing.html"
+    },
+    
+  
+    
+    "template_type": "keyboard",
+
```
Comment (navneetzz) [inline share/goodie/cheat_sheets/json/Bing]: @mahavirrathod here also additional space

File: share/goodie/cheat_sheets/json/Bing [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    "section_order": [  
+        "Navigate 2D maps",
+        "Navigate 3D maps in first-person mode",
+        "Navigate Photosynths"
+       
+    ],
+
```
Comment (navneetzz) [inline share/goodie/cheat_sheets/json/Bing]: Here the same

File: share/goodie/cheat_sheets/json/Bing [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+            },
+            {
+                "key": "Z",
+                "val": "Move back one photo"
+            },
+             {
+                "key": "I",
+                "val": "Move to the next photo in"
+            },
+             {
```
Comment (navneetzz) [inline share/goodie/cheat_sheets/json/Bing]: @mahavirrathod here indentation is not same as rest of the files

File: share/goodie/cheat_sheets/json/Bing [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+        "Navigate Photosynths": [
+            {
+                "key": "[SPACEBAR]",
+                "val": "Move to the next photo"
+            },
+            {
+                "key": "Z",
+                "val": "Move back one photo"
+            },
+             {
```
Comment (navneetzz) [inline share/goodie/cheat_sheets/json/Bing]: @mahavirrathod here indentation is not same as rest of the files

File: share/goodie/cheat_sheets/json/Bing [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+            
+             {
+                "key": "+",
+                "val": "Zoom in"
+            },
+             {
+                "key": "-",
+                "val": "Zoom out"
+            }
+
```
Comment (navneetzz) [inline share/goodie/cheat_sheets/json/Bing]: @mahavirrathod here indentation of brackets is not same as rest of the files.

File: share/goodie/cheat_sheets/json/Bing [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+                "val": "Zoom in"
+            },
+             {
+                "key": "-",
+                "val": "Zoom out"
+            }
+            
+        ]
+     
+
```
Comment (navneetzz) [inline share/goodie/cheat_sheets/json/Bing]: here also extra space

**Comment by alohaas** [comment]
@mahavirrathod, thanks again for working on building a new Bing Cheat Sheet! You’ve been doing a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in Bing and optimizing search workflow.

When you're ready to move on, please: 

- Create a new Instant Answer for searches you do related to Bing or optimizing search workflow

OR

- Test live Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

**Comment by mintsoft** [comment]
@mahavirrathod  I'm going to close this PR for now; feel free to reopen it whenever you like :+1:

**Pull request closed without merging**